source/javascripts/embertimes-test-ids.js

```diff
@@ -0,0 +1,27 @@
+$(function(){
+  if ($('[data-embertimes-page]').length > 0) {
+    applyEmberTimesTestIds();
+  }
+
+
+   function applyEmberTimesTestIds() {
+     let sectionTitles = $('[data-embertimes-post-body] h2');
+     sectionTitles.attr('data-embertimes-section-title', true);
```

Comment:
Do we need this line? It seems, in line 12, you are setting a unique ID to section title.

Reply:
You're right, this one can be removed - thank you for the catch :+1:

Reply:
```suggestion
```